File: pinyougou-sellergoods-interface/src/main/java/com/rl/sellergoods/service/TypeTemplateService.java
package com.rl.sellergoods.service;

import com.rl.pojo.TbTypeTemplate;
import entity.PageResult;

/**
 * @Author Ren
 * @Description //TODO 模板管理
 * @Date 2018/10/8 17:28
 * @Param
 * @return
 **/
public interface TypeTemplateService {
    //查询
    PageResult findPage(TbTypeTemplate typeTemplate, int page, int rows);

    void add(TbTypeTemplate typeTemplate);

    TbTypeTemplate findOne(Long id);

    void update(TbTypeTemplate typeTemplate);

    void delete(long[] ids);
}

File: pinyougou-manager-web/src/main/webapp/js/service/typeTemplateService.js
app.service('typeTemplateService',function($http){

    this.search=function (page, rows, searchEntity) {
        return $http.post("../typeTemplate/search.do?page="+page+"&rows="+rows,searchEntity);
    }
    //增加
    this.add=function (entity) {
        return $http.post("../typeTemplate/add.do",entity);
    }
    //修改
    this.update=function (entity) {
        return $http.post("../typeTemplate/update.do",entity);
    }
    //查询一个
    this.findOne=function (id) {
        return $http.post("../typeTemplate/findOne.do?id="+id);
    }
    //删除
    this.del=function (ids) {
        return new $http.get("../typeTemplate/delete.do?ids="+ids);
    }
});